src/pages/community/members.js

```diff
@@ -1,26 +1,60 @@
 
 const options = [
-  { label: "All Members", value: "all", color: `${theme.linkColor}`, isFixed: true},
-  { label: "Active Members", value: "active", color: `${theme.linkColor}`, isFixed: true},
-  { label: "Inactive Members", value: "inactive", color: `${theme.menuColor}`, isFixed: true},
-  { label: "Maintainers", value: "maintainers", color: `${theme.linkColor}`, isFixed: true},
-  { label: "MeshMates", value: "meshmates", color: `${theme.linkColor}`, isFixed: true}
+  { label: <option disabled className="category">BADGES&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;</option>, value: "", color: `${theme.linkColor}`,isFixed: true},
+  { label: <option className="allOptions">&ensp;&emsp;&nbsp;&nbsp;&nbsp;Community</option>, value: "community", color: `${theme.linkColor}`, isFixed: true},
+  { label: <option className="allOptions">&ensp;&emsp;&nbsp;&nbsp;&nbsp;Landscape</option>, value: "lanscape", color: `${theme.linkColor}`, isFixed: true},
+  { label: <option className="allOptions">&ensp;&emsp;&nbsp;&nbsp;&nbsp;Image Hub</option>, value: "imagehub", color: `${theme.linkColor}`, isFixed: true},
+  { label: <option className="allOptions">&ensp;&emsp;&nbsp;&nbsp;&nbsp;Meshery</option>, value: "meshery", color: `${theme.linkColor}`, isFixed: true},
+  { label: <option className="allOptions">&ensp;&emsp;&nbsp;&nbsp;&nbsp;Meshery Operator</option>, value: "mesheryoperator", color: `${theme.linkColor}`, isFixed: true},
+  { label: <option className="allOptions">&ensp;&emsp;&nbsp;&nbsp;&nbsp;Service Mesh Performance</option>, value: "smp", color: `${theme.linkColor}`, isFixed: true},
+  { label: <option className="allOptions">&ensp;&emsp;&nbsp;&nbsp;&nbsp;GetNighthawk</option>, value: "getnighthawk", color: `${theme.linkColor}`, isFixed: true},
+  { label: <option disabled className="category">STATUS&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;</option>, value: "", color: `${theme.linkColor}`, isFixed: true},
+  { label: <option className="allOptions">&ensp;&emsp;&nbsp;&nbsp;&nbsp;All Members</option>, value: "all", color: `${theme.linkColor}`, isFixed: true},
+  { label: <option className="allOptions">&ensp;&emsp;&nbsp;&nbsp;&nbsp;Active Members</option>, value: "active", color: `${theme.linkColor}`, isFixed: true},
+  { label: <option className="allOptions">&ensp;&emsp;&nbsp;&nbsp;&nbsp;Inactive Members</option>, value: "inactive", color: `${theme.menuColor}`, isFixed: true},
+  { label: <option disabled className="category">ROLE&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;</option>, value: "", color: `${theme.linkColor}`, isFixed: true},
+  { label: <option className="allOptions">&ensp;&emsp;&nbsp;&nbsp;&nbsp;Maintainers</option>, value: "maintainers", color: `${theme.linkColor}`, isFixed: true},
+  { label: <option className="allOptions">&ensp;&emsp;&nbsp;&nbsp;&nbsp;MeshMates</option>, value: "meshmates", color: `${theme.linkColor}`, isFixed: true}
 ];
 
+var style = document.createElement("style");
```

Comment:
This won't work (as seen from failed build) as gatsby follows server-side rendering in production.